pkg/devicetwin/dtmanager/membership_test.go

```diff
@@ -0,0 +1,352 @@
+/*
+Copyright 2018 The KubeEdge Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dtmanager
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/golang/mock/gomock"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/mocks/beego"
+	"github.com/kubeedge/kubeedge/pkg/common/dbm"
```

Comment:
group golang, kubeedge and vendor packages and seperate them with a blank line.

Reply:
separated golang,kubeedge and vendor packages.

Reply:
It should be something like this
```
import (
	"encoding/json"
	"errors"
	"testing"
	"time"

	"github.com/golang/mock/gomock"

	"github.com/kubeedge/kubeedge/beehive/pkg/core"
	"github.com/kubeedge/kubeedge/beehive/pkg/core/context"
	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
	"github.com/kubeedge/kubeedge/mocks/beego"
	"github.com/kubeedge/kubeedge/mocks/beehive"
	"github.com/kubeedge/kubeedge/pkg/common/dbm"
	"github.com/kubeedge/kubeedge/pkg/common/util"
	"github.com/kubeedge/kubeedge/pkg/metamanager/dao"
)
```

Reply:
Modified.